core/src/Cuda/Kokkos_Cuda_Instance.cpp

```diff
@@ -313,6 +313,7 @@ void CudaInternal::initialize(int cuda_device_id, cudaStream_t stream) {
 
   enum { WordSize = sizeof(size_type) };
 
+#ifndef KOKKOS_IMPL_TURN_OFF_CUDA_HOST_INIT_CHECK
```

Comment:
So the intention is you configure with `-CMAKE_CXX_FLAGS="-D KOKKOS_IMPL_TURN_OFF_CUDA_HOST_INIT_CHECK"`?

Reply:
Yeah, sorry, Christian and I discussed that on Slack